srcpkgs/sumo/template

```diff
@@ -1,18 +1,20 @@
 # Template file for 'sumo'
 pkgname=sumo
-version=1.5.0
-revision=4
+version=1.12.0
+revision=1
 build_style=cmake
-python_version=3
 hostmakedepends="libgdal-tools pkg-config swig python3-setuptools"
-makedepends="python3-devel ffmpeg-devel fox-devel gl2ps-devel libgdal-devel libxerces-c-devel osg-devel proj-devel"
+makedepends="python3-devel ffmpeg-devel fox-devel gl2ps-devel libgdal-devel
+ libxerces-c-devel osg-devel proj-devel"
 short_desc="SUMO (Simulation of Urban MObility) and included applications"
 maintainer="Warrior Graph <marques@larces.uece.br>"
 license="EPL-2.0"
 homepage="https://sumo.dlr.de"
 distfiles="https://sumo.dlr.de/releases/${version}/sumo-src-${version}.tar.gz"
```

Comment:
```suggestion
homepage="https://sumo.dlr.de"
changelog="https://raw.githubusercontent.com/eclipse/sumo/800cb422fa4b9107a1d799d73fc19dbdd7f2cdab/docs/web/docs/ChangeLog.md"
distfiles="https://sumo.dlr.de/releases/${version}/sumo-src-${version}.tar.gz"
```

Reply:
with that commit hash? i would use https://raw.githubusercontent.com/eclipse/sumo/master/docs/web/docs/ChangeLog.md

Reply:
I didn't notice I copied a commit hash, master will be better :)